build/upgrade/Dockerfile

```
@@ -71,7 +74,8 @@ RUN for pg_version in $UPGRADE_PG_VERSIONS; do \
                 postgresql${pg_version}-contrib \
                 postgresql${pg_version}-server \
                 pgaudit${pg_version} \
-                pgnodemx${pg_version} ; \
+                pgnodemx${pg_version} \
+				orafce_${pg_version//.} ; \
```

```suggestion
				orafce_${pg_version} ; \
```
Probably not necessary to make this change, I'm just wondering if that `//.` on line 67 and 78 is strictly necessary?

---

The `//.` has been removed.